{"name": "test_delete_added_request", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_case_details.TestCaseDetails object at 0x0DBAC6B0>\n\n    def test_delete_added_request(self):\n        driver = self.driver\n        self.login()\n        case_details = CaseDetails(driver)\n        home = HomePage(driver)\n        home.click_law_suit_page()\n        case_details.request_tab_entry()\n>       case_details.delete_added_request()\n\ntest_case_details.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Pages.CaseDetailsPage.CaseDetails object at 0x0DBACAD0>\n\n    def delete_added_request(self):\n        table = self.driver.find_elements_by_xpath(self.directive_table_xpath) # tbody\n        for tds in table:\n            tds = tds.find_elements_by_tag_name(\"td\")\n            for trs in tds:\n                deneme = trs.find_elements_by_tag_name(\"tr\")\n>               print(deneme[0].text)\nE               IndexError: list index out of range\n\n..\\Pages\\CaseDetailsPage.py:180: IndexError"}, "start": 1568630963578, "stop": 1568630992033, "uuid": "18a44743-5302-4571-a1a7-ec80d35a8d79", "historyId": "812199f097f3799c533dced022272d50", "testCaseId": "546de6c208ed1c45d32a4ab8f8a947c2", "fullName": "test_case_details.TestCaseDetails#test_delete_added_request", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_case_details"}, {"name": "subSuite", "value": "TestCaseDetails"}, {"name": "host", "value": "EVT01066NB"}, {"name": "thread", "value": "16968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_details"}]}